### ISSUE TITLE
After moving a folder into a new one: "server replied: Conflict" on another client

### ISSUE THREAD

**Issue by tuxayo** [owncloud/core#20526]
**edit 2016-11-12**: I can't reproduce the issue but see https://github.com/owncloud/core/issues/20526#issuecomment-260016437 for another case that happened recently.

---

I had a folder named "musique libre".

On the machine D (desktop) I moved it to a folder called "music"

Latter I checked the client and it asked if I would like to sync "music" as it was > 500MB. I said yes.

Latter I turned on the machine L (laptop) and the client asked if I would like to sync "music" as it was > 500MB. I said yes.

Now, on the machine L, the "musique libre" folder has not been moved to "music" and I got this message:

```
Error downloading https://example.org/owncloud/remote.php/webdav/musique libre/to_sort/Chiptune - server replied: Conflict
```

On the two machines and the web interface. There is not duplication of the folder (as expected with a conflict) so I can't solve the conflict.
The machine L is not out of sync regarding the folder "music" and "musique libre" is still there.

What could I do to help finding the cause of this bug? And how to simply get the machine L back in sync?

### Server configuration

**Operating system**:
Arch Linux

**Web server:**
nginx 1.8.0

**Database:**
PostgreSQL 9.3.5

**PHP version:**
5.6.15

**ownCloud version:** 8.2.0

**Updated from an older ownCloud or fresh install:**
Updated from a 8.0 install

**Are you using encryption:** yes
### Client configuration

Arch Linux
Client: 2.0.2

**Comment by tuxayo** [comment]
Got back to a consitent state by deleting the account on the client and adding it back.
The old folder was still here and some of the files were still in it instead of the new folder.

**Comment by PVince81** [comment]
Has this ever happened again with 8.2.2 ?

**Comment by tuxayo** [comment]
I only have one client for now (until July). So I can't try to reproduce that issue.
Furthermore, I have other more important issues of files refusing to sync so I don't think I'll take the time to help for this one until some time.

**Comment by PVince81** [comment]
Has the problem appeared again lately ?

**Comment by tuxayo** [comment]
Thanks for the followup. It turns out that I still have one machine so I can't test it.
In 10 days I should have my second machine back. So I should be able to retry and report back here within the next month.

**Comment by JetUni** [comment]
I just had this happen to me and had to remove the account and add it back to fix it. In my case, I renamed a folder on one computer, then deleted it on that same computer and that's when I had problems.

### Server configuration

**Operating system:**
Ubuntu 16.04 LTS

**Web server:**
Apache 2.4.18

**Database:**
mySQL 5.7.15

**PHP version:**
PHP 7.0.8

**ownCloud version:** 9.1.0

**Are you using encryption:** yes

### Client configuration

Windows
Client: 2.2.3

**Comment by tuxayo** [comment]
@PVince81 I can't reproduce it. Thanks for your patience.

**Comment by tuxayo** [comment]
@JetUni It's the same error message but the scenario is different so these issues might no have anything in common.

Can you reproduce your issue?

**Comment by nemobis** [comment]
The same is happening to me after I renamed a directory on 9.1.1.

**Comment by tuxayo** [comment]
@nemobis Can you reproduce the issue? If yes, what are the minimal steps? If not, what do you remember having done to trigger it?

**Comment by nemobis** [comment]
I don't know exactly, and I'm not going to try multiplying the bug since 
it blocks syncronisation for all users in my org.

There was an error with sync of A uploaded by X in directory B; the only 
fix is usually to delete the entire directory and reupload from scratch, 
and I did so; the error continued; I renamed B to B.old, the error 
continued on B.old; I unshared B.old and then deleted B.old from the web 
interface; now X uploads A in B, buy my client claims there is a 
conflict and refuses to upload the rest of the directory (which I can't 
see on the web interface).

I've closed ownCloud for a few hours and now the sync completed without 
errors (perhaps the other user went offline); my copy of the file has 
been moved to A_conflict-20161111-182715.
